imports/ui/cdp/CDPs.jsx

```diff
@@ -33,9 +60,89 @@ export default class CDPs extends Component{
+                    </NavLink>
+                </NavItem>
+                <NavItem>
+                    <NavLink
+                        className={classnames({ active: this.state.cdpActiveTab === 'cdp-bnb' })}
+                        onClick={() => { this.toggleTab('cdp-bnb', 'bnb-a'); }}
+                    >
+                        <span className="cdp-logo"><img src="/img/BNB-symbol.svg" className="symbol-img" /> BNB</span>
+
+                    </NavLink>
+                </NavItem>
+                <NavItem>
+                    <NavLink
+                        className={classnames({ active: this.state.cdpActiveTab === 'cdp-hard' })}
+                        onClick={() => { this.toggleTab('cdp-hard', 'hard-a'); }}
+                    >
+                        <span className="cdp-logo"><img src="/img/HARD-symbol.svg" className="symbol-img" /> HARD</span>
+                    </NavLink>
+                </NavItem> 
+                <NavItem>
+                    <NavLink
+                        className={classnames({ active: this.state.cdpActiveTab === 'cdp-btcb' })}
+                        onClick={() => { this.toggleTab('cdp-btcb', 'btcb-a'); }}
+                    >
+                        <span className="cdp-logo"><img src="/img/BTCB-symbol.svg" className="symbol-img" /> BTCB</span>
+                    </NavLink>
+                </NavItem> 
+                <NavItem>
+                    <NavLink
+                        className={classnames({ active: this.state.cdpActiveTab === 'cdp-xrpb' })}
+                        onClick={() => { this.toggleTab('cdp-xrpb', 'xrpb-a'); }}
+                    >
+                        <span className="cdp-logo"><img src="/img/XRP-symbol.svg" className="symbol-img" /> XRPB</span>
+                    </NavLink>
+                </NavItem>   
+                <NavItem>
+                    <NavLink
+                        className={classnames({ active: this.state.cdpActiveTab === 'cdp-busd' })}
+                        onClick={() => { this.toggleTab('cdp-busd', 'busd-a'); }}
+                    >
+                        <span className="cdp-logo"><img src="/img/BUSD-symbol.svg" className="symbol-img" /> BUSD</span>
+                    </NavLink>
+                </NavItem>    
+            </Nav>
+            <TabContent activeTab={this.state.cdpActiveTab}>
+                <TabPane tabId="cdp-kava">
+                    <Switch>
+                        <Route exact path="/cdps" component={this.CDPList} />
+                    </Switch>
+                </TabPane>
+                <TabPane tabId="cdp-bnb">
+                    <Switch>
+                        <Route exact path="/cdps" component={this.CDPList} />
+                    </Switch>
+                </TabPane>
+                <TabPane tabId="cdp-hard">
+                    <Switch>
+                        <Route exact path="/cdps" component={this.CDPList} />
```

Comment:
What's the usage of creating multiple routes with the same route here?

Reply:
Ohh! Fixed that!